background.js

```diff
@@ -54,16 +65,19 @@
         function (details) {
             let url = details.url;
             if (isEnabled && localStorage.getItem(url)) {
-                let newUrl = url.replace(URL_REGEX, URL_REPLACEMENT);
+                let newUrl = url.replace(URL_REGEX, getReplacementURL());
                 let matches = URL_REGEX.exec(sender.url);
-                if (matches[1] in SPECIAL_CASES) {
-                    newUrl = URL_REPLACEMENT.replace('$1', '') + SPECIAL_CASES[matches[1]];
+                if (matches[1] == pyVersion){
+                    return {};
+                }
+                if (matches[2] in SPECIAL_CASES) {
+                    newUrl = getReplacementURL().replace('$2', '') + SPECIAL_CASES[matches[2]];
                 }
                 return {redirectUrl: newUrl};
             }
         },
         {
-            urls: ['*://docs.python.org/2*'],
+            urls: ['*://docs.python.org/[23]*'],
```

Comment:
```suggestion
            urls: ['*://docs.python.org/*'],
```
This is an invalid wildcard, causing the cache not to work at all. See for reference:
* https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns
* https://developer.chrome.com/extensions/match_patterns

Reply:
Got it, thanks - I didn't have a good way to check that